### PR TITLE
feat: add fuel testnet contract address

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: pages/price-feeds/contract-addresses/starknet.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Price Feed Contract Addresses on Starknet
 
-Pyth is currently deployed on Starknet Testnet
+Pyth is currently deployed on Starknet Testnet.
```
Comment (cctdaniel) [inline pages/price-feeds/contract-addresses/starknet.mdx:3]: drive-by

**Review by ali-bahjati — APPROVED** [review]
